### ISSUE TITLE
Add "Timer tick sound" to features list

### ISSUE THREAD

**Issue by schmich** [schmich/marinara#143]
- `_locales/en/messages.json`, `chrome_web_store_description`
- `README.md`
- Chrome Web Store

**Comment by gitpet** [comment]
Thanks for the ticker sound! I love using it. I just wish it was possible to reduce the timer tick sound volume independently from the other sound notifications.

Thanks,
Peter

**Comment by schmich** [comment]
Thanks for the feedback, Peter.

Which tick sound(s) are you using? Which sound notification(s) are you using?

There were requests for volume controls for the sound notifications, too, but I pushed back on those. Instead, I normalized all the alarm sounds to an average volume that precludes the need for custom volume controls.

I want to push back on timer tick volume controls as well. Would it be acceptable to just reduce the overall volume of the tick sounds relative to the sound notifications? I know this might be hard to get right, but it would keep things simpler.

I am not entirely against custom volume controls, and ultimately I might add them, it's just that I am *obligated* to work against more complexity, more settings, more code, and more bloat in this extension. There are another two dozen features to add, and I want to keep things lean where possible.

Let me know what you think.
